neqo-common/src/qlog.rs

```diff
@@ -29,21 +30,53 @@ pub struct NeqoQlogShared {
 }
 
 impl NeqoQlog {
-    /// Create an enabled `NeqoQlog` configuration.
+    /// Create an enabled `NeqoQlog` configuration backed by a file.
     ///
     /// # Errors
     ///
-    /// Will return `qlog::Error` if cannot write to the new log.
-    pub fn enabled(
-        mut streamer: QlogStreamer,
-        qlog_path: impl AsRef<Path>,
+    /// Will return `qlog::Error` if it cannot write to the new file.
+    pub fn enabled_with_file(
+        mut qlog_path: PathBuf,
+        role: Role,
+        title: Option<String>,
+        description: Option<String>,
+        file_prefix: impl Display,
     ) -> Result<Self, qlog::Error> {
+        qlog_path.push(format!("{file_prefix}.sqlog"));
```

Comment:
Previously we would use `.qlog` on the server and `.sqlog` on the client. As far as I can tell, the `s` stands for _streaming_. Thus I argue both should use `.sqlog`. Am I missing something?

https://quicwg.org/qlog/draft-ietf-quic-qlog-main-schema.html#name-qlog-to-json-text-sequences

Reply:
No, you are correct.